testing/spectest/shared/phase0/ssz_static/ssz_static.go

```diff
@@ -25,66 +25,70 @@ type SSZRoots struct {
 // RunSSZStaticTests executes "ssz_static" tests.
 func RunSSZStaticTests(t *testing.T, config string) {
 	require.NoError(t, utils.SetConfig(t, config))
-
 	testFolders, _ := utils.TestFolders(t, config, "phase0", "ssz_static")
 	for _, folder := range testFolders {
-		innerPath := path.Join("ssz_static", folder.Name(), "ssz_random")
-		innerTestFolders, innerTestsFolderPath := utils.TestFolders(t, config, "phase0", innerPath)
+		modePath := path.Join("ssz_static", folder.Name())
+		modeFolders, _ := utils.TestFolders(t, config, "phase0", modePath)
 
-		for _, innerFolder := range innerTestFolders {
-			t.Run(path.Join(folder.Name(), innerFolder.Name()), func(t *testing.T) {
-				serializedBytes, err := util.BazelFileBytes(innerTestsFolderPath, innerFolder.Name(), "serialized.ssz_snappy")
-				require.NoError(t, err)
-				serializedSSZ, err := snappy.Decode(nil /* dst */, serializedBytes)
-				require.NoError(t, err, "Failed to decompress")
-				object, err := UnmarshalledSSZ(t, serializedSSZ, folder.Name())
-				require.NoError(t, err, "Could not unmarshall serialized SSZ")
+		for _, modeFolder := range modeFolders {
+			innerPath := path.Join(modePath, modeFolder.Name())
+			innerTestFolders, innerTestsFolderPath := utils.TestFolders(t, config, "phase0", innerPath)
 
-				rootsYamlFile, err := util.BazelFileBytes(innerTestsFolderPath, innerFolder.Name(), "roots.yaml")
-				require.NoError(t, err)
-				rootsYaml := &SSZRoots{}
-				require.NoError(t, utils.UnmarshalYaml(rootsYamlFile, rootsYaml), "Failed to Unmarshal")
+			for _, innerFolder := range innerTestFolders {
+				t.Run(path.Join(folder.Name(), innerFolder.Name()), func(t *testing.T) {
```

Comment:
```suggestion
				t.Run(path.Join(modeFolder.Name(), folder.Name(), innerFolder.Name()), func(t *testing.T) {
```

Can we also add this? Otherwise we are seeing duplicate tests with the same names in the testlogs